### ISSUE TITLE
This version does not work.

### ISSUE THREAD

**Issue by MichaelPittingerSr** [KazuCocoa/alexa-tide-pooler#1]
Doesn't work....  fails to retrieve from NOAA.

the version in the alexa app store works....

The remote endpoint could not be called, or the response it returned was invalid. is the response in testing from this:

{
  "session": {
    "sessionId": "SessionId.dfc8deb9-6911-48fe-af6c-09fc83b4b4a6",
    "application": {
      "applicationId": "amzn1.ask.skill.afe454f4-1364-47ac-b322-9476dbdf71cc"
    },
    "attributes": {},
    "user": {
      "userId": "amzn1.ask.account.AFJJAO7JJ2VZZTRHR2LELOBLMATRXF6JIWTPNGF77OZ42U6GNQAAHBLBXG6SWMXFOGC3AK4WN27CKAPIEEVNU7AHNQOGQODUKW2TLYPNRZEB6L3FE3KFKDFQVYB3RDE5QKDY6PKFUC2G7E2SHDA2WCBAET3I7EPOSOE5ZLZ5HJ6URBNRIVOHGQXE3M3KSA43T2JL3D4PBH5F6WQ"
    },
    "new": true
  },
  "request": {
    "type": "IntentRequest",
    "requestId": "EdwRequestId.fa33c430-4c6c-459d-b6c2-ab427012fdb4",
    "locale": "en-US",
    "timestamp": "2017-03-28T14:38:34Z",
    "intent": {
      "name": "OneshotTideIntent",
      "slots": {
        "State": {
          "name": "State"
        },
        "City": {
          "name": "City",
          "value": "Seattle"
        },
        "Date": {
          "name": "Date",
          "value": "2017-04-01"
        }
      }
    }
  },
  "version": "1.0"
}

**Comment by harshidwd** [comment]
is it resolved?

**Comment by KazuCocoa** [comment]
No, since I have no time to resolve this issue since this is just my hobby project :p

**Comment by waninggibbous** [comment]
Hi, I think I know what the problem is.  The http request needs to use https; also, there are a couple of  invalid references to NOAARequestObj within the calling function.
